packages/pwa/pwa-kit-config.json

```diff
@@ -0,0 +1,8 @@
+{
+    "app": {
+        "url": {
+          "locale": "query_param",
```

Comment:
what does this configuration mean?

does it move the locale from url path to query parameter?

like `/en-GB/cart` v.s. `/cart?locale=en-GB`?

Reply:
Yes